test/tasks/status_test.rb

```diff
@@ -45,7 +45,7 @@ class RedisStatusTest < ActiveSupport::TestCase
 
 class KafkaStatusTest < ActiveSupport::TestCase
   test 'kafka:status fails without a kafka connection' do
-    TestProducer.stubs(:deliver_message).raises(Kafka::ConnectionError)
+    ApplicationProducer.stubs(:kafka).raises(Kafka::ConnectionError)
```

Comment:
Something about the way I was stubbing `Kafka::Client.any_instance` messed up Jenkins :thinking: I have no clue why, but this alternate stub seems to have fixed the issue.

Reply:
What the :duck:  dude ...